.github/workflows/e2e.yaml

```diff
@@ -1,13 +1,17 @@
 name: e2e
 on:
   pull_request:
+    paths-ignore:
+      - '**/*.md'
```

Comment:
ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet